ramls/packages.raml

```diff
@@ -1,169 +1,111 @@
   post:
     description: Create a custom package
     headers:
       Content-Type:
         example: application/vnd.api+json
     body:
       application/vnd.api+json:
-        properties:
-          data:
-            description: Needed because of JSON API
-            type: object
-            required: true
-            properties:
-              type: string
-              attributes:
-                description: Needed because of JSON API
-                type: object
-                required: true
-                properties:
-                  name:
-                    description: Name of the custom package to be created
-                    type: string
-                    example: My test package
-                    required: true
-                  contentType:
-                    description: Content type of the custom package to be created
-                    type: string
-                    enum: ["Aggregated Full Text", "Abstract and Index", "E-Book", "E-Journal", "Print", "Online Reference", "Unknown"]
-                    example: Unknown
-                    required: true
-                  customCoverage:
-                    description: Coverage dates of the custom package to be created
-                    type: customCoverage
-                    required: false
-        example: !include examples/packages/packages_post_request.json
+          description: Create a custom package
+          type: packagePostRequest
+          example:
+            strict: false
+            value: !include examples/packages/packages_post_request.json
     responses:
       200:
         description: OK
         body:
           application/vnd.api+json:
-            example: !include examples/packages/packages_post_200_response.json
+            description: The server has successfully fulfilled the POST request.
+            type: package
+            example:
+              strict: false
+              value: !include examples/packages/packages_post_200_response.json
       400:
         description: Bad Request
         body:
           application/vnd.api+json:
-              example: !include examples/packages/packages_post_400_response.json
+            type: jsonapiError
+            example:
+              strict: false
+              value: !include examples/packages/packages_post_400_response.json
+      422:
+        description: Unprocessable Entity
+        body:
+          application/vnd.api+json:
+            type: jsonapiError
+            example:
+              strict: false
+              value: !include examples/packages/packages_post_422_response.json
   /{packageId}:
+    displayName: Get package by package Id
+    description: Instance of a package given packageId
+    uriParameters:
+        packageId:
+          description: identifier of the package formed from Provider Id and Package Id (see example)
+          type: string
+          example: 583-4345
```

Comment:
Do we want to make packageId required: true?

Reply:
it's a uri parameter. w/o it it doesn't make sense at all
we do not define uri params as required explicitly in other ramls. see providerId from providers.raml for example:
```
/{providerId}:
    displayName: Get provider by provider Id
    description: Instance of a provider given providerId
    uriParameters:
      providerId:
        description: Provider Id of provider to get
        type: string
    get:
      description: Get provider given providerId
      is: [includable]
      responses:
        200:
..
```